website-1.x/blog/2021-05-11-releasing-docusaurus-beta.md

```diff
@@ -0,0 +1,84 @@
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, beta]
+# image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release the first **Docusaurus 2 beta**, after a **very long alpha period**.
+
+We believe Docusaurus 2 is ready for **mainstream adoption**!
+
+<!--truncate-->
+
+## Docusaurus adoption
+
+**Don't be afraid of the beta tag!**
+
+Docusaurus 2 is **widely adopted** and growing fast:
+
+![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/trend.png)
+
+Our new [showcase](https://docusaurus.io/showcase) page allows you to **filter Docusaurus sites by features**: get inspired by real-world production sites with a similar use-case!
+
+Don't miss our [favorite](https://docusaurus.io/showcase?tags=favorite) sites, they all **stand out** with something unique:
+
+[![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/favorites.png)](https://docusaurus.io/showcase?tags=favorite)
+
+## Why was Docusaurus v2 in alpha for so long?
+
+The first alpha release [v2.0.0-alpha.0](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.0) was **2 years ago** 😅.
+
+It was definitively a very long and unusual alpha period.
+
+Docusaurus 2 has reached **feature parity** with Docusaurus 1 with **[i18n](https://docusaurus.io/blog/2021/03/09/releasing-docusaurus-i18n)**, and it has been **successfully adopted** by many Docusaurus sites (see [i18n showcase](https://docusaurus.io/showcase?tags=i18n)).
+
+We're now confident that the **core features** of Docusaurus 2 are **stable enough** to be in beta.
+
+## What are the goals of the beta?
+
+We will use the beta phase to **improve our theming system**.
+
+We want to make it:
+
+- **easier to implement a custom theme**, including for ourselves. We want to provide [multiple official themes](https://github.com/facebook/docusaurus/issues/3522) (including [Tailwind CSS](https://github.com/facebook/docusaurus/issues/2961)) for a long time.
+
+- **safer to extend an existing theme**: it can be painful to upgrade a **highly customized** Docusaurus site, as customizations can **conflict with internal changes**. We need to make the theme public API surface more explicit, and make it clear what is safe to customize.
+
+We will build a better **theming infrastructure** and refactor the classic theme to use it.
+
+If you don't customize your Docusaurus site much, you can consider Docusaurus stable.
+
+## What's new?
+
+In case you missed it, we recently shipped two major improvements:
+
+- [Auto-generated sidebars](https://docusaurus.io/docs/sidebar#sidebar-item-autogenerated): no need to maintain a `sidebars.js` file anymore!
+- [Webpack 5 / PostCSS 8](https://github.com/facebook/docusaurus/issues/4027): persistent caching significantly speeds up **rebuild time**!
+
+## What's next?
+
+We will continue to **fix bugs** and implement the **most wanted features**, some of them are:
```

Comment:
```suggestion
Shipping the official 2.0 release! 

To get to 2.0 RTM, we will continue to **fix bugs** and implement the **most wanted features**, including:
```

Reply:
I have no idea what `2.0 RTM` stands for, so please either: expand the abbreviation in full, or use write something like `To get to the official 2.0 release of Docusaurus, we will...`

Reply:
also not familiar with RTM, had to google it with  "RTM + software" to find what this is. More familiar with "GA" but I don't think many users will understand either